openedx/frontend-app-course-authoring
# fix: studio home UI bugs

JIRA Ticket: [TNL-11054](https://2u-internal.atlassian.net/browse/TNL-11054)

> 1. Update re-run side bar section 1 description to use local time instead of UTC
> 2. Make the error message in new course and course rerun forms font larger.
> 3. Update colors used for error/denied, pending, and in-progress/unrequested card alerts
>   i. error/denied - `bg-danger-100` and `text-gray-700`
>   ii. pending - `bg-warning-100` and `text-gray-700`
>   iii. in-progress - `bg-light-100` and `text-gray-700`
> 4. Add New library button that links to legacy library tab. When LA MFE is enabled it will direct to the create page  
> 5. Fix spacing and font size when using org search
> 6. Update create course redirect to point at legacy outline page

### Testing

Create Library
1. Navigate to the [home page](http://localhost:2001/home)
2. Navigate to [Django admin](http://localhost:18010/admin/) waffle flags and enable `contentstore.split_library_on_studio_dashboard`
3. Click the "New library" button
4. Should redirect to the legacy studio home with a focus on the library tab
5. The address bar should read `http://localhost:18010/home_library`
6. Navigate back to the studio home mfe
7. Disable `contentstore.split_library_on_studio_dashboard`
8. Click the "New library" button
9. Should redirect to the legacy studio home with a focus on the library tab
10. The address bar should read `http://localhost:18010/home#libraries-tab`
11. Navigate to [Django admin](http://localhost:18010/admin/) waffle flags and enable `studio.library_authoring_mfe`
12. Click the "New library" button
13. Should redirect to the library authoring mfe
14. The address bar should read `http://localhost:3001/create`

Create course
1. Click "New course"
2. Fill out all of the information
3. Click "Create course"
4. Should be redirected to outline for new course

Rerun course
1. Click "re-run course"
2. Fill out all of the information
3. Click "Create re-run"
4. Should be redirected to outline for new course re-run
